src/components/Reports/ReportEdit.tsx

```diff
@@ -193,8 +193,14 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
       snackbar.toastWarning(strings.REPORT_BACK_WARNING);
     });
   }, [snackbar]);
-
   const hasEmptyRequiredFields = (iReport: Report) => {
+    const emptyWorkerField = (location: any) => {
```

Comment:
Use this helper function to check for worker fields being filled out. This was missed before somehow.